### ISSUE TITLE
Categories (TG) in config not working

### ISSUE THREAD

**Issue by Disonantemus** [Beastwick18/nyaa#29]
Works good with Nyaa, but not TG, this is my TG section in my config:

~~~ocaml
[source.torrentgalaxy]
base_url = "https://torrentgalaxy.to/"
default_sort = "Date"
default_sort_dir = "Desc"
default_filter = "NoFilter"
default_category = "HD TV"
default_search = "1080p x265"
~~~

I did use [Category Values from here](https://github.com/Beastwick18/nyaa/wiki/Torrent-Galaxy#categories).
When I launch nyaa, starts with TG source, but only showing "AllCategories"
I did try with `default_category = "Sports"` and `default_category = "Bollywood"` with no success.

---

### Workaround

Looking at the code in [torrent_galaxy.rs](https://github.com/Beastwick18/nyaa/blob/a5eecbb9db35ac2d760704e44d38588eaa463091/src/source/torrent_galaxy.rs#L687), I found the following line (#660):
~~~ocaml
"TV" => {41 => ("HdT", "TV HD", "HdTV", source.tgx.cat.tv_hd);
~~~

I did try "HdTV" in the config, and now **it works!** (maybe is not checking "TV HD" and "HdT", only the last one):

~~~ocaml
default_category = "HdTV"
~~~

---

~~~ocaml
$ nyaa -v
nyaa v0.9.1

$ neofetch
distro: Arch Linux x86_64
kernel: 6.6.22-1-lts
shell: bash 5.2.32
~~~

**Comment by Beastwick18** [comment]
The values are the wiki are wrong, for both TG is sukebei. I had a macro to write them, but I must've selected the wrong column.

The line
```rust
"TV" => {41 => ("HdT", "TV HD", "HdTV", source.tgx.cat.tv_hd);
```
indicates that the icon is "HdT", the display name is "TV HD", and the config value is "HdTV".

To make this easier, I'll make the display name and even the icon name an "alias" for the config value, since they are all distinct. So any of "HdT", "TV HD", or "HdTV" should work.

I'll go ahead and update the values on the wiki too.

**Comment by Beastwick18** [comment]
Fixed in ffb5010
